cypress/cypress.config.js

```diff
@@ -0,0 +1,48 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      console.log('Starting script');
+      return getLocations(`${config.baseUrl}/sitemap.xml`).then(urls => {
+        console.log(urls);
+        config.env.URLS = urls;
+        return config;
+      });
+    },
+    baseUrl: "http://localhost:7000",
```

Comment:
If we want to run it locally, we may need to make it configurable (to use `dev.local` for example :p)

Reply:
This is working with `npm run ci` because `dev.local:7000` is not provided in the `udata-front-e2e.cfg`.